olap-examples/aws-setup/OLAP_AWS.md

```diff
@@ -0,0 +1,83 @@
+# AWS Spark Cluster Setup
+
+To create a spark cluster in AWS, the following can be used from command line.
+
+```bash
+emr_name="olap-spark-cluster-aws"
+region=us-east-1
+instance_type="m7a.2xlarge"
+num_workers=20
+num_cores_per_instance=$(aws ec2 describe-instance-types --instance-types "$instance_type" --query "InstanceTypes[0].VCpuInfo.DefaultVCpus" --output text --region $region)
+spark_executor_cores=$((num_workers * num_cores_per_instance))
+olap_jar="s3://<path>/aerospike-graph-olap-<version>.jar"
+properties_file_uri="s3://<path>/aerospike-graph-olap.properties"
+step_name="olap-step"
+log_uri="s3://<path>/logs/"
+
+# Recommend using 75% of memory per core, which for an 8g/core machine, is 6g
+memory=6g 
```

Comment:
That's the case for GCP's `n2d-highmem-8` not AWS `m7a.2xlarge` which has 32GB of RAM so its 4g/core, which means it should be `memory=3g` if we go with `m7a.2xlarge`?

Reply:
Probably we should change our aws suggestion here, this was an oversight on my part.